sockeye/vocab.py

```diff
@@ -279,7 +296,8 @@ def load_or_create_vocabs(source_paths: List[str],
         vocab_source = load_or_create_vocab(source_path, source_vocab_path, num_words_source, word_min_count_source,
                                             pad_to_multiple_of=pad_to_multiple_of)
         vocab_target = load_or_create_vocab(target_path, target_vocab_path, num_words_target, word_min_count_target,
-                                            pad_to_multiple_of=pad_to_multiple_of)
+                                            pad_to_multiple_of=pad_to_multiple_of,
```

Comment:
The case of a shared vocabulary above is not covered. Could you pass `num_pointers` also in line 289 above?

Reply:
Added!